src/main/java/org/creekservice/kafka/test/perf/implementations/DevHarrelImplementation.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2023 Creek Contributors (https://github.com/creek-service)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.creekservice.kafka.test.perf.implementations;
+
+import static org.creekservice.kafka.test.perf.testsuite.SchemaSpec.DRAFT_2019_09;
+import static org.creekservice.kafka.test.perf.testsuite.SchemaSpec.DRAFT_2020_12;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import dev.harrel.jsonschema.Dialects;
+import dev.harrel.jsonschema.SpecificationVersion;
+import dev.harrel.jsonschema.Validator;
+import java.awt.Color;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.Set;
+import org.creekservice.kafka.test.perf.model.TestModel;
+import org.creekservice.kafka.test.perf.testsuite.AdditionalSchemas;
+import org.creekservice.kafka.test.perf.testsuite.SchemaSpec;
+
+@SuppressWarnings("FieldMayBeFinal") // not final to avoid folding.
+public class DevHarrelImplementation implements Implementation {
+
+    private static final MetaData METADATA =
+            new MetaData(
+                    "json-schema (dev.harrel)",
+                    "dev.harrel",
+                    Language.Java,
+                    Licence.MIT,
+                    Set.of(DRAFT_2020_12, DRAFT_2019_09),
+                    "https://github.com/harrel56/json-schema",
+                    new Color(235, 54, 172));
+
+    private ObjectMapper mapper = JsonMapper.builder().build();
+
+    @Override
+    public MetaData metadata() {
+        return METADATA;
+    }
+
+    @Override
+    public JsonValidator prepare(
+            final String schema, final SchemaSpec spec, final AdditionalSchemas additionalSchemas) {
+
+        final Validator validator = validator(spec);
+        /* Parse remotes eagerly and ignore errors from unknown specs */
+        additionalSchemas
+                .remotes()
+                .forEach(
+                        (uri, remote) -> {
+                            try {
+                                validator.registerSchema(uri, remote);
+                            } catch (Exception e) {
+                                /* ignore */
+                            }
+                        });
+        final URI schemaUri = validator.registerSchema(schema);
+
```

Comment:
Rather than parsing _all_ remotes each time, does the validator not have a way of loading them on demand?  

While this won't _currently_ affect benchmark results, as the prepare statement isn't timed, users may look at this implementation as a reference.  Good practice here is to be encouraged :)

In the future, there could be a benchmark measuring the time it takes to prepare, i.e. parse, the schemas.

Reply:
Not parsing remotes eagerly will affect benchmark - it would get parsed during validation logic (it's not parsed during registration process, references are resolved only if needed during validation). But I can change it - I checked it and it looked like it wasn't a significant performance hit

Reply:
If that's the case, then I have no issue with you parsing the in prepare.  

Unless there was some way you could force your impl to parse them?  I'm guess trying to validate an empty document in `prepare` wouldn't cause the reference to be loaded?

Reply:
Currently I cannot think of a way for achieving that, empty document wouldn't trigger that (in most cases). But I tested out the performance for both solutions and honestly I didn't notice any difference - so it's really fine